src/utils.ts

```diff
@@ -23,7 +23,7 @@ export function getDescription(incident: Incident) {
   for (const update of incident.incident_updates) {
     const time = new Date(update.created_at);
 
-    description += `\n**${pascalCase(update.status)}** - _${time.toUTCString()}_\n${update.body}\n`;
+    description += `\n**${pascalCase(update.status)}** - <t:${Math.floor(time / 1000)}:F>\n${update.body}\n`;
```

Comment:
Can we have both the relative timestamp and the UTC? For very old incis etc. the relative timestamp will become rather unhelpful

Reply:
This is not the relative timestamp, you are thinking of `:R>` at the end.
<img width="1226" alt="image" src="https://user-images.githubusercontent.com/40232557/196774327-f191ec96-6668-43d4-8f8f-48d6e89120ce.png">

Reply:
Ah! My bad -- maybe relative + long would be useful for folks?

Reply:
```suggestion
    description += `\n**${pascalCase(update.status)}** - <t:${Math.floor(time / 1000)}:F> (<t:${Math.floor(time / 1000)}:R>)\n${update.body}\n`;
```